.github/workflows/extrinsic-ordering-check-from-bin.yml

```diff
@@ -69,7 +74,7 @@ jobs:
 
       - name: Stop our local node
         run: |
-          pkill polkadot-parachain
+          pkill node-bin
```

Comment:
Ah it hurts having to use it here.
How about:
```
BIN=node-bin
BIN_PATH=./tmp/$BIN
```
that gives you all the VARS you need to NOT type `node-bin` anywhere but at the beginning when defining the VARS.